lib/result.ex

```diff
@@ -12,11 +12,9 @@ defmodule PropCheck.Result do
     :gen_server.start_link({ :local, __MODULE__ }, __MODULE__, [], [])
   end
   def stop do
-    try do
-      :gen_server.call(__MODULE__, :stop)
-    catch
-      _ -> :ok
-    end
+    :gen_server.call(__MODULE__, :stop)
```

Comment:
I am not sure that I like this and to be honest I didn't know that this possible at all. For a minimal function like this is perfectly ok, but in a larger context I would like to know what the scope of the try-catch block. If this rule only applies to entire functions, then I am fine.

Reply:
I checked https://github.com/rrrene/credo/issues/224 and the Elixir docs. So it only makes sense for a function that starts with a try block closes with the corresponding catch etc block. So, I think this is ok.